component-library/src/app/gallery/QA/naseer/naseer.component.ts

```diff
@@ -19,6 +19,21 @@ export class NaseerComponent implements OnInit {
     formGroup: this.form
   };
 
+  qaButton : IIconButtonComponentConfig = {
+    id: "qa_test_button",
+    size: 'extraSmall',
+    category: 'custom',
+    icon:  {
+      class: 'fa-solid fa-mustache',
+      color: 'var(--primary-text)'
```

Comment:
Make an enum of all the tokens and place it in a file called ds-tokens.ts inside the shared/interfaces folder. Then make the interface for this (icon.color, which should be changed to icon.colour [unless color is used everywhere else] accept keyof typeof YourEnum || string)